### ISSUE TITLE
Upgrade to connect v3

### ISSUE THREAD

**Issue by aduh95** [AveVlad/gulp-connect#237]
Hello,

I got the folling message when installing the package with yarn:

```
warning gulp-connect > connect@2.30.2: connect 2.x series is deprecated
```

Is there anything holding us back to use connect v3?

**Comment by JonasGao** [comment]
+1

**Comment by maaizelahi** [comment]
+1

**Comment by nickpape-msft** [comment]
+1 this seems to be blocking our upgrade to Node 8.

**Comment by rejas** [comment]
Should be resolved with v5.4.0 that just got published.
